src/OrbitGl/CaptureViewElementTest.cpp

```diff
@@ -8,20 +8,35 @@
 
 namespace orbit_gl {
 
-class UnitTestCaptureViewElement : public CaptureViewElement {
+class UnitTestCaptureViewLeafElement : public CaptureViewElement {
  public:
-  explicit UnitTestCaptureViewElement(CaptureViewElement* parent, TimeGraph* time_graph,
-                                      Viewport* viewport, TimeGraphLayout* layout,
-                                      int child_count = 0)
+  explicit UnitTestCaptureViewLeafElement(CaptureViewElement* parent, TimeGraph* time_graph,
+                                          Viewport* viewport, TimeGraphLayout* layout)
+      : CaptureViewElement(parent, time_graph, viewport, layout) {}
+
+  [[nodiscard]] float GetHeight() const override { return 10; }
+
+ private:
+  [[nodiscard]] std::unique_ptr<orbit_accessibility::AccessibleInterface>
+  CreateAccessibleInterface() override {
+    return nullptr;
+  }
+};
+
+class UnitTestCaptureViewContainerElement : public CaptureViewElement {
+ public:
+  explicit UnitTestCaptureViewContainerElement(CaptureViewElement* parent, TimeGraph* time_graph,
+                                               Viewport* viewport, TimeGraphLayout* layout,
+                                               int children_to_create = 0)
       : CaptureViewElement(parent, time_graph, viewport, layout) {
-    for (int i = 0; i < child_count; ++i) {
+    for (int i = 0; i < children_to_create; ++i) {
       children_.emplace_back(
-          std::make_unique<UnitTestCaptureViewElement>(this, time_graph, viewport, layout));
+          std::make_unique<UnitTestCaptureViewLeafElement>(this, time_graph, viewport, layout));
     }
   }
 
   [[nodiscard]] float GetHeight() const override { return 0; }
```

Comment:
So, the container has 0 of height and 10 the leaf. Shouldn't be the sum of children's heights?

Reply:
That's a good point... it's not needed for this test, but I've changed it to sum up the height of children nevertheless for consistency.